orttraining/orttraining/python/training/ortmodule/_io.py

```diff
@@ -573,12 +575,12 @@ def _add_input(name, input_value, onnx_graph, onnx_graph_input_names):
     )
 
 
-def parse_outputs_for_onnx_export_and_extract_schema(module, inputs, kwargs):
+def parse_outputs_for_onnx_export_and_extract_schema(module, inputs, kwargs, log_level):
     # Perform a forward call to grab outputs
     output_names = None
     output_dynamic_axes = None
     is_deepcopy = False
-    with torch.no_grad():
+    with torch.no_grad(), suppress_os_stream_output(log_level=log_level):
```

Comment:
Which output are we trying to suppress here?

Reply:
When we try to print something in the autograd Function (PythonOp in ORT), during log level warning, the log output between ORT and PyTorch is not comparable now because ORTModule here print logs for a PyTorch fw run before exporting (BTW, exporting log is suppressed already). 

Suppressing it, then the logs are comparable. This introduce a big confusion to me when I am trying to understand the diff between ORT and PyTorch.